core/src/main/scala/cats/TraverseFilter.scala

```diff
@@ -22,6 +22,7 @@
 package cats
 
 import cats.data.State
+import cats.implicits.toTraverseOps
```

Comment:
Can we write this without the ops import? Current practice is try to avoid circular dependencies between syntax and typeclasses.

Reply:
I tried to do manual `sequence` for this operation. WDYT?